### PR TITLE
force cgroup v1

### PR DESCRIPTION
**Pull request by calvix**
Add feature to force cgroups v1 for AWS to specific node pools

### CODE REVIEW

File: service/internal/cloudconfig/tcnp.go [hunk 1]
```diff
@@ -92,6 +94,36 @@ func (t *TCNP) NewTemplates(ctx context.Context, obj interface{}) ([]string, err
...
+		md = list.Items[0]
+	}
+
+	var forceCGroupsV1 bool
+	{
+		if _, ok := md.Annotations[annotation.NodeForceCGroupsV1]; ok {
+			forceCGroupsV1 = true
+		} else {
+			forceCGroupsV1 = false
+		}
```
Comment (njuettner) [inline service/internal/cloudconfig/tcnp.go:124]: We could drop those lines, but only if you prefer it

**Review by njuettner — APPROVED** [review]

File: service/internal/cloudconfig/tcnp.go [hunk 2]
```diff
@@ -92,6 +94,36 @@ func (t *TCNP) NewTemplates(ctx context.Context, obj interface{}) ([]string, err
...
+	}
```
Comment (AverageMarcus) [inline service/internal/cloudconfig/tcnp.go:125]:
```suggestion
	_, forceCGroupsV1 := md.Annotations[annotation.NodeForceCGroupsV1]
```

Comment (AverageMarcus) [reply service/internal/cloudconfig/tcnp.go:125]: Minor suggestion to save lines. I don't mind either way.

Comment (calvix) [reply service/internal/cloudconfig/tcnp.go:125]: I was thinking about this, but it might be harder to read, but no strong feelings

**Pull request merged**